gradle.properties

```diff
@@ -4,6 +4,7 @@
 #
 
 version=1.0.0
+systemProp.bwc.version=1.3.1
```

Comment:
please make sure it's covered in plugin release guide

Reply:
Will update release guide

Reply:
Updated.